vector/src/main/java/im/vector/app/features/call/webrtc/WebRtcCall.kt

```diff
@@ -770,12 +775,49 @@ class WebRtcCall(
         return currentCaptureFormat
     }
 
-    fun startSharingScreen() {
-        // TODO. Will be handled within the next PR.
+    fun startSharingScreen(videoCapturer: VideoCapturer) {
```

Comment:
Have you tried to extract the screen sharing capabilities into a dedicated component to avoid adding new responsabilities to `WebRtcCall`?

Reply:
I don't think it is easily possible. This class has many local fields. We will start implementing Element Call soon, so we probably won't invest in this class more.